src/grip/specifications/grip.md

```diff
@@ -0,0 +1,31 @@
+# Grip
+
+Use it to indicate that a part of the screen is reactive to slide touch events.
+
+<!-- STORY -->
+
+## Usage
+
+`TextComponents` are exported individually.
+
+```js
+import Grip from '@blablacar/ui-library/build/grip'
+;<Grip
+  onSlideUp={() => {
+    /* ... */
+  }}
+  onSlideDown={() => {
+    /* ... */
+  }}
+>
+  <TheVoice>What is your route?</TheVoice>
+</Grip>
+```
+
+## Props
```

Comment:
If you write the story on .MDX, it will have properties for free ;)
Please refeer to `ItemBigData` component for an exemple or any of my open PRs

 https://github.com/blablacar/ui-library/pulls/gabrielagabriel

Reply:
Thanks!